syndeo-template/src/main/java/com/google/cloud/syndeo/SyndeoTemplate.java

```diff
@@ -82,9 +82,9 @@ public interface Options extends PipelineOptions, DataflowPipelineOptions {
           Set.of("table", "createDisposition", "writeDisposition"),
           // New names:
           "beam:schematransform:org.apache.beam:bigquery_storage_write:v1",
-          Set.of("instanceId", "tableId", "keyColumns", "projectId", "appProfileId"),
+          Set.of(),
           "beam:schematransform:org.apache.beam:bigquery_storage_read:v1",
-          Set.of("table"),
+          Set.of(),
```

Comment:
Is the intention here to not check the parameters against a Set() anymore and let everything through?

Reply:
yes, correct!